index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="Header">Josué</header>
```

Comment:
- [ ] Please make sure to use the required font-family for this section: 
![image](https://user-images.githubusercontent.com/52635378/170834172-1bbcc88b-ad2c-40d5-81ff-d797742a0bef.png)